src/worker.rs

```diff
@@ -104,6 +108,18 @@ fn list_files(logger: &Logger, name: &str, recursive: bool) -> Result<Vec<File>,
                                             err,
                                         )
                                     })?);
+
+                                    let get_sha = Command::new("svn")
+                                        .arg("propget")
+                                        .arg("git-commit")
+                                        .arg("--revprop")
+                                        .arg("-r")
+                                        .arg(format!("{}", revision.unwrap()))
+                                        .arg(format!("{}/{}/trunk", ORGANIZATION_ROOT, query_name))
+                                        .output()
+                                        .expect("Failed to get sha of revision");
```

Comment:
Should never except this, then we'll panic and break the whole thread. Instead, match on it.